delete-closed-pr-docs/action.yaml

```diff
@@ -39,7 +39,7 @@ runs:
 
         closed_docs_versions=()
         for docs_version in ${{ steps.get-docs-versions.outputs.docs-versions }}; do
-          pr_number=$(echo "$docs_version" | grep -oP "pr-\K([0-9]+)")
+          pr_number=$(echo "$docs_version" | grep -oP "pr-\w+")
```

Comment:
Oh sorry, we had to extract the number only.

```
$ echo "pr-47" | grep -oP "pr-\K([0-9]+)"
47
```

Reply:
But it seems the current way works correctly... :thinking: 

```
$ echo "pr-47 [latest]" | grep -oP "pr-\K([0-9]+)"
47
```

Reply:
@KeisukeShima Ah, you have to change L32.

```
        echo ::set-output name=docs-versions::$(mike list | grep "pr-")
```

Reply:
Thank you! I changed the script 2cc8867

Reply:
@kenji-miyake Do you usually test the workflow? If so, what is your procedure?

Reply:
It depends. In this case, I think just testing locally is enough, I think.

Reply:
In this PR, I've temporary changed the version to the PR branch.
https://github.com/autowarefoundation/autoware.core/pull/4
https://github.com/autowarefoundation/autoware.core/pull/4/commits/9f6971ae03818f097a5b21dfe27ed5c26888c1d3